data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,436 @@
+			`	"Everything else should be ready on our end soon enough, if you can just handle that pick-up and delivery we'll be ok."`
+			`	You assure her you can handle a simple pick-up.`
+				accept
+	on stopover
+		dialog `You have the furnishings the Hai ordered. It's time to return them to <destination>.`
+	on complete
+		event "role-play station active" 12
+		event "news: hai video clip debunked"
+		log `Of all the hilariously mundane things that could trip up a deception, it turned out to be Hai-made furniture. Fortunately you were on hand to save the day by acquiring a large shipment of human-made furniture from Prime in order to complete the station.`
+		conversation
+			`By the time you return you are shocked to notice that the station seems to be rapidly approaching a completed state. Some four thousand people have already arrived, though some are still on board ships docked at the station. Once you register your delivery amidst the station chaos it's only a few minutes later that you get a "thank you" message from Sayari. She strongly recommends that you hang around Hai space for the time being while events are ongoing, with a cryptic "P.S. you may get a message from a friend of mine soon."`
+
+event "role-play station active"
+	system "Ultima Thule"
+		government "Neutral"
+		object
+			sprite star/f0
+			period 10
+		object
+			sprite planet/desert5
+			distance 200
+			period 22.3607
+		object
+			sprite planet/europa-b
+			distance 403.41
+			period 64.056
+		object
+			sprite planet/desert7-b
+			distance 824.9
+			period 187.302
+		object
+			sprite planet/gas1
+			distance 1706.99
+			period 557.553
+			object
+				sprite planet/ice8-b
+				distance 216
+				period 11.1964
+		object "Wormhole Alpha"
+			sprite planet/wormhole-syndicate-ad
+				"frame rate" 1
+				"random start frame"
+			distance 4208.63
+			period 2158.5
+			object Mountaintop
+				sprite "planet/station hai eight geocoris"
+				distance 315
+				period 31.1964
+	planet Mountaintop
+		government "Neutral"
+		attributes station tourism
+		landscape land/station5
+		description `This brand new station is built with wide spacious interiors, though that is largely because it was built for having expansion room inside it to accommodate the Hai's preferred geometric development pattern and long-term perspectives. At the moment there is a number of hai pretending to be humans in costume, and a number of volunteer humans from Hai space helping to make up the act. A powerful holoprojection system is projecting an 'anomaly' into space nearby as a novelty. Some 'investigators' following rumors of the Hai are being entertained at most any time.`
```

Comment:
```suggestion
		description `This brand new station is built with wide spacious interiors for later expansion, a staple of Hai building. At the moment, there is a number of Hai pretending to be humans in costume and volunteer humans from Hai space helping to make up the act. A powerful holoprojection system is projecting an 'anomaly' into space nearby as a novelty. It seems to reasonably entertain some 'investigators' following rumors of the Hai.`
```
The explanation for the large interiors felt too academic/explanatory/long winded(?). I think I cut too much out, but I can't seem to come up with anything longer to land it in a nice "middle ground" for length. Also, the `at most any time` just feels really hard to read and understand, so the rephrase is an alternative I came up with.

Reply:
I'd like to include the geometric construction aspect, since it's only infrequently referred to. other than that I like it though.

Reply:
Edited the original b/c that's easier than making a new suggestion on mobile. How does it sound now?